### ISSUE TITLE
Name collision

### ISSUE THREAD

**Issue by KottV** [jpcima/spectacle#7]
Hi!

Unfortunately there is screenshoting tool from KDE with the same name: https://github.com/jpcima/spectacle

**Comment by jpcima** [comment]
In the commit a47fd22, I am renaming the target name, such that there won't be any file conflict with other packages.

Since the LV2 plugin URL doesn't change, this will probably not break any sessions.

The official package name becomes `spectacle-analyzer`. It will be used starting with the next release.

**Comment by trebmuh** [comment]
Will you rename this repo too?

**Comment by jpcima** [comment]
I don't consider it necessary, maybe.
This annoys me more, given there exist already a published link out there.
GH is going to create a redirect, but I don't know if it's lasting / how long.

**Comment by trebmuh** [comment]
Yep, agreed that it isn't strictly mandatory. My question was more about a consistency approach. Not a big deal.

All good then!

**Comment by luzpaz** [comment]
Maybe worth just renaming since KDE spectacle is very prominent. 

Spectacle analyzer isn't a bad option:  
https://libreav.org/software/spectacle-analyzer
